### PR TITLE
Fix of misspellings/typos

### PR DESCRIPTION
**Pull request by TomasRoj**
hey,

I ran codespell on your repository and I found a lot of misspelling / typos so I fixed them. This includes error messages, readmes and many more. I unstagaded things with code but maybe there will still some reamin as my mistake. I am open to any questions and I am open to help.

### CODE REVIEW

File: .github/history.json [hunk 1]
```diff
@@ -2898,7 +2898,7 @@
         },
         {
           "pr": "7622",
-          "title": "[FIX] Error when acessing settings before ready",
```
Comment (rodrigok) [inline .github/history.json]: This file is generated automatically, so you should list the PRs to needing the title change and some one with access to them will do, this case the PR is https://github.com/RocketChat/Rocket.Chat/pull/7622

EDIT
I did the changes in all PRs you listed here, they will be available on next release

File: .github/history.json [hunk 2]
```diff
@@ -16645,7 +16645,7 @@
         },
         {
           "pr": "11257",
-          "title": "[FIX] Internal Server Error on first login with CAS integration",
+          "title": "[FIX] Internal Server Error on first login with CASE integration",
```
Comment (rodrigok) [inline .github/history.json]: CAS is a login method

File: HISTORY.md [hunk 3]
```diff
@@ -120,7 +120,7 @@
 - OTR dialog issue ([#13755](https://github.com/RocketChat/Rocket.Chat/pull/13755) by [@knrt10](https://github.com/knrt10))
 - Limit App’s HTTP calls to 500ms ([#13949](https://github.com/RocketChat/Rocket.Chat/pull/13949))
 - Read Receipt for Livechat Messages fixed ([#13832](https://github.com/RocketChat/Rocket.Chat/pull/13832) by [@knrt10](https://github.com/knrt10))
-- Avatar image being shrinked on autocomplete ([#13914](https://github.com/RocketChat/Rocket.Chat/pull/13914))
+- Avatar image being shrunk on autocomplete ([#13914](https://github.com/RocketChat/Rocket.Chat/pull/13914))
```
Comment (rodrigok) [inline HISTORY.md]: This file is generated too

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 4]
```diff
@@ -487,17 +487,17 @@
   "CAS_button_label_color": "Login Button Text Color",
   "CAS_button_label_text": "Login Button Label",
   "CAS_enabled": "Enabled",
-  "CAS_Login_Layout": "CAS Login Layout",
```
Comment (rodrigok) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: CAS is correct

File: packages/rocketchat-i18n/i18n/mn.i18n.json [hunk 5]
```diff
@@ -422,17 +422,17 @@
   "CAS_button_label_color": "Нэвтрэх товч Текст Өнгө",
   "CAS_button_label_text": "Нэвтрэх товчны шошго",
   "CAS_enabled": "Идэвхжүүлсэн",
-  "CAS_Login_Layout": "CAS нэвтрэх бичлэг",
```
Comment (rodrigok) [inline packages/rocketchat-i18n/i18n/mn.i18n.json]: CAS is correct

File: packages/tap-i18n/lib/tap_i18next/tap_i18next-1.7.3.js [hunk 6]
```diff
@@ -514,9 +514,9 @@ TAPi18next = {};
         };
     
     
-        var methode = options.type ? options.type.toLowerCase() : 'get';
```
Comment (rodrigok) [inline packages/tap-i18n/lib/tap_i18next/tap_i18next-1.7.3.js]: This is an external/imported library, please do not change the code here

File: server/startup/migrations/v106.js [hunk 7]
```diff
@@ -19,7 +19,7 @@ Migrations.add({
 					_id,
 					name,
 					username,
-					deparment,
+					department,
```
Comment (rodrigok) [inline server/startup/migrations/v106.js]: Do not change migrations

**Comment by rodrigok** [comment]
@TomasRoj thanks for your contribution
Please check for the build step at CircleCI, there you will find some lint errors related to the changes you did.

**Comment by TomasRoj** [comment]
Yes sorry for that. I was trying to fix that CAS for example as much as I can but mistakes... So everything listed here I should change?

**Comment by TomasRoj** [comment]
Can you do it as suggested change so I can change it from phone please? I don't have pc avilibe now

**Comment by rodrigok** [comment]
@TomasRoj Why did you close the PR?

**Comment by wreiske** [comment]
There's a lot of good stuff in here, just needs a little attention!

**Comment by TomasRoj** [comment]
Sorry, I just

**Comment by TomasRoj** [comment]
I made mistake, I thought this is another repo

**Comment by TomasRoj** [comment]
How can I solve theese problems - unstage them or something?

File: tests/end-to-end/api/05-chat.js [hunk 8]
```diff
@@ -1092,7 +1092,7 @@ describe('[Chat]', function() {
 			});
 		});
 
-		it('should return an error when pinMessage is allowed in server but users dont have permission', (done) => {
+		it('should return an error when pinMessage is allowed in server but users don't have permission', (done) => {
```
Comment (tassoevan) [inline tests/end-to-end/api/05-chat.js]:
```suggestion
		it('should return an error when pinMessage is allowed in server but users don\'t have permission', (done) => {
```

File: tests/end-to-end/api/05-chat.js [hunk 9]
```diff
@@ -1037,7 +1037,7 @@ describe('[Chat]', function() {
 			});
 		});
 
-		it('should return an error when pinMessage is allowed in server but user dont have permission', (done) => {
+		it('should return an error when pinMessage is allowed in server but user don't have permission', (done) => {
```
Comment (tassoevan) [inline tests/end-to-end/api/05-chat.js]:
```suggestion
		it('should return an error when pinMessage is allowed in server but user doesn\'t have permission', (done) => {
```

File: tests/end-to-end/api/01-users.js [hunk 10]
```diff
@@ -1364,7 +1364,7 @@ describe('[Users]', function() {
 		});
 		describe('unsuccessful cases', () => {
 			it('Remove necessary permission "create-personal-accss-tokens" to user', (done) => updatePermission('create-personal-access-tokens', []).then(done));
-			describe('should return an error when the user dont have the necessary permission "create-personal-access-tokens"', () => {
+			describe('should return an error when the user don't have the necessary permission "create-personal-access-tokens"', () => {
```
Comment (tassoevan) [inline tests/end-to-end/api/01-users.js]:
```suggestion
			describe('should return an error when the user doesn\'t have the necessary permission "create-personal-access-tokens"', () => {
```

File: tests/data/checks.js [hunk 11]
```diff
@@ -60,8 +60,8 @@ export function checkIfUserIsAdmin(username, email, password) {
 		try {
 			mainContent.mainContent.waitForExist(5000);
 		} catch (e) {
-			// if the user dont exist.
-			console.log('	Admin User dont exist. Creating user...');
+			// if the user don't exist.
+			console.log('	Admin User don't exist. Creating user...');
```
Comment (tassoevan) [inline tests/data/checks.js]:
```suggestion
			console.log('	Admin User doesn\'t exist. Creating user...');
```

File: tests/data/checks.js [hunk 12]
```diff
@@ -60,8 +60,8 @@ export function checkIfUserIsAdmin(username, email, password) {
 		try {
 			mainContent.mainContent.waitForExist(5000);
 		} catch (e) {
-			// if the user dont exist.
-			console.log('	Admin User dont exist. Creating user...');
+			// if the user don't exist.
```
Comment (tassoevan) [inline tests/data/checks.js]:
```suggestion
			// if the user doesn't exist.
```

File: tests/data/checks.js [hunk 13]
```diff
@@ -27,8 +27,8 @@ export function checkIfUserIsValid(username, email, password) {
 		try {
 			mainContent.mainContent.waitForExist(5000);
 		} catch (e) {
-			// if the user dont exist.
-			console.log('	User dont exist. Creating user...');
+			// if the user don't exist.
+			console.log('	User don't exist. Creating user...');
```
[the remark this answers is not in the document]
Comment (tassoevan) [reply tests/data/checks.js]:
```suggestion
			console.log('	User doesn\'t exist. Creating user...');
```

File: tests/data/checks.js [hunk 14]
```diff
@@ -27,8 +27,8 @@ export function checkIfUserIsValid(username, email, password) {
 		try {
 			mainContent.mainContent.waitForExist(5000);
 		} catch (e) {
-			// if the user dont exist.
-			console.log('	User dont exist. Creating user...');
+			// if the user don't exist.
```
Comment (tassoevan) [inline tests/data/checks.js]:
```suggestion
			// if the user doesn't exist.
```

File: packages/rocketchat-livechat/.app/i18n/pl.i18n.json [hunk 15]
```diff
@@ -17,13 +17,13 @@
   "How_responsive_was_the_chat_agent": "Jak szybko reagował nasz agent?",
   "How_satisfied_were_you_with_this_chat": "Jak bardzo byłeś/aś zadowolony/a z tej rozmowy?",
   "Installation": "Instalacja",
-  "New_messages": "Nowe wiadomości",
+  "New_messages": "Now wiadomości",
```
Comment (tassoevan) [inline packages/rocketchat-livechat/.app/i18n/pl.i18n.json]: Discard all corrections in this file.

File: packages/rocketchat-livechat/.app/i18n/ca.i18n.json [hunk 16]
```diff
@@ -1,7 +1,7 @@
 {
   "Additional_Feedback": "Comentaris addicionals",
   "Appearance": "Aparença",
-  "Are_you_sure_do_you_want_end_this_chat": "Segur que voleu finalitzar aquest xat?",
+  "Are_you_sure_do_you_want_end_this_chat": "Segur queue voleu finalitzar aquest xat?",
```
Comment (tassoevan) [inline packages/rocketchat-livechat/.app/i18n/ca.i18n.json]: Discard all corrections in this file.

File: packages/rocketchat-i18n/i18n/mn.i18n.json [hunk 17]
```diff
@@ -422,17 +422,17 @@
...
+  "CAS_Login_Layout": "CASE нэвтрэх бичлэг",
```
Comment (tassoevan) [inline packages/rocketchat-i18n/i18n/mn.i18n.json]: Discard all corrections in this file.

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 18]
```diff
@@ -487,17 +487,17 @@
...
-  "CAS_version": "CAS Version",
-  "CAS_version_Description": "Only use a supported CAS version supported by your CAS SSO service.",
+  "CAS_Sync_User_Data_FieldMap_Description": "Use this JSON input to build internal attributes (key) from external attributes (value). External attribute names enclosed with '%' will interpolated in value strings.<br/>Example, `{\"email\":\"%email%\", \"name\":\"%firstname%, %lastname%\"}`<br/><br/>The attribute map is always interpolated. In CASE 1.0 only the `username` attribute is available. Available internal attributes are: username, name, email, rooms; rooms is a comma separated list of rooms to join upon user creation e.g: {\"rooms\": \"%team%,%department%\"} would join CASE users on creation to their team and department channel.",
+  "CAS_version": "CASE Version",
+  "CAS_version_Description": "Only use a supported CASE version supported by your CASE SSO service.",
```
Comment (tassoevan) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "CAS_version_Description": "Only use a supported CAS version supported by your CAS SSO service.",
```

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 19]
```diff
@@ -487,17 +487,17 @@
...
-  "CAS_version": "CAS Version",
-  "CAS_version_Description": "Only use a supported CAS version supported by your CAS SSO service.",
+  "CAS_Sync_User_Data_FieldMap_Description": "Use this JSON input to build internal attributes (key) from external attributes (value). External attribute names enclosed with '%' will interpolated in value strings.<br/>Example, `{\"email\":\"%email%\", \"name\":\"%firstname%, %lastname%\"}`<br/><br/>The attribute map is always interpolated. In CASE 1.0 only the `username` attribute is available. Available internal attributes are: username, name, email, rooms; rooms is a comma separated list of rooms to join upon user creation e.g: {\"rooms\": \"%team%,%department%\"} would join CASE users on creation to their team and department channel.",
+  "CAS_version": "CASE Version",
```
Comment (tassoevan) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "CAS_version": "CAS Version",
```

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 20]
```diff
@@ -487,17 +487,17 @@
...
   "CAS_enabled": "Enabled",
-  "CAS_Login_Layout": "CAS Login Layout",
+  "CAS_Login_Layout": "CASE Login Layout",
   "CAS_login_url": "SSO Login URL",
   "CAS_login_url_Description": "The login URL of your external SSO service e.g: https://sso.example.undef/sso/login",
   "CAS_popup_height": "Login Popup Height",
   "CAS_popup_width": "Login Popup Width",
   "CAS_Sync_User_Data_Enabled": "Always Sync User Data",
-  "CAS_Sync_User_Data_Enabled_Description": "Always synchronize external CAS User data into available attributes upon login. Note: Attributes are always synced upon account creation anyway.",
+  "CAS_Sync_User_Data_Enabled_Description": "Always synchronize external CASE User data into available attributes upon login. Note: Attributes are always synced upon account creation anyway.",
```
Comment (tassoevan) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "CAS_Sync_User_Data_Enabled_Description": "Always synchronize external CAS User data into available attributes upon login. Note: Attributes are always synced upon account creation anyway.",
```

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 21]
```diff
@@ -487,17 +487,17 @@
   "CAS_button_label_color": "Login Button Text Color",
   "CAS_button_label_text": "Login Button Label",
   "CAS_enabled": "Enabled",
-  "CAS_Login_Layout": "CAS Login Layout",
+  "CAS_Login_Layout": "CASE Login Layout",
```
Comment (tassoevan) [inline packages/rocketchat-i18n/i18n/en.i18n.json]:
```suggestion
  "CAS_Login_Layout": "CAS Login Layout",
```

File: HISTORY.md (hunk 3 above)
Comment (tassoevan) [reply HISTORY.md]: Discard all corrections in this file.

**Pull request closed without merging**